packages/uauth/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "@web3-onboard/uauth",
+  "version": "1.0.0",
```

```suggestion
  "version": "2.0.0-alpha.1",
```
Wince this is v2 of onboard we are starting all of our packages at v2 for uniformity. The alpha flag will trigger a pre-release NPM publish that both our team and yours can test.